src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.jsx

```diff
@@ -8,7 +8,7 @@ import styled from '@emotion/styled';
 
 import {inputStyles} from 'app/styles/input';
 import DropdownMenu from 'app/components/dropdownMenu';
-import InlineSvg from 'app/components/inlineSvg';
+import {IconCalendar} from 'app/icons/iconCalendar';
```

Comment:
We generally import from 'app/icons' I think, should we try to be consistent here?

Reply:
Yep - I can change it

Reply:
We seem to import using both ways pretty evenly, or rather, we only use  `app/icons` when importing multiple icons. This is a pretty easy refactor, so no need to change it atm.